<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpperComputer_HK</class>
 <widget class="QMainWindow" name="UpperComputer_HK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpperComputer_HK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="RcvdArea">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SendArea">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>181</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label1">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>串口号：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Cb_ComName">
                 <property name="font">
                  <font>
                   <family>Monotype Corsiva</family>
                   <italic>true</italic>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>波特率：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Cb_Baudrate">
                 <item>
                  <property name="text">
                   <string>1200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>4800</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>9600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>14400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>38400</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>43000</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>57600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>76800</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>115200</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label3">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>停止位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Cb_StopPos">
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1.5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label4">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>数据位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Cb_DataPos">
                 <item>
                  <property name="text">
                   <string>5</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>6</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>7</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>校验位：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Cb_CheckPos">
                 <property name="font">
                  <font>
                   <family>Segoe Script</family>
                  </font>
                 </property>
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Odd</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Even</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_OpenCom">
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QCheckBox" name="Ckb_HexDisplay">
             <property name="text">
              <string>16进制显示</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="Ckb_ChRcvd">
             <property name="text">
              <string>文本显示</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QCheckBox" name="Ckb_HexSend">
           <property name="text">
            <string>16进制发送</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Ckb_ChSend">
           <property name="text">
            <string>文本发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QCheckBox" name="Ckb_AutoSend">
           <property name="text">
            <string>定时发送 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LE_AutoSendTime">
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QPushButton" name="Btn_Send">
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_StopSend">
             <property name="text">
              <string>停止发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="Btn_PIDMode">
           <property name="text">
            <string>PID模式</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="Btn_ClearRcvd">
             <property name="text">
              <string>清除接收</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_ClearSend">
             <property name="text">
              <string>清除发送</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label4_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>P:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_P">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label4_3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>I:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_I">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label4_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>D:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_D">
         <property name="text">
          <string>0.0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
